.gitignore

```diff
@@ -129,4 +129,7 @@ setup.log
 _opam/
 
 # for Nix
-result/
\ No newline at end of file
+result/
+/.envrc.local
+/result
```

Comment:
I'm confused---why isn't `/result` covered by `result/` already?

Reply:
`git` sees the symlink as a file not a directory. So I think we want either `/result` or `result`. Since the `.gitignore` file is at the root of the repo we ought to just use `result`. I'll make that update now.